docs/getting-started/ios/index.md

```diff
@@ -3,3 +3,56 @@
 
 This page describes how to get started building an application with Warp components.
 
+## 1. Integrate Warp
+
+A guide on how to integrate Warp into your project.
+
+### Swift Package Manager
+
```

Comment:
Do we need to give more explanations here? Is this the only thing a user needs to do in order to integrate Warp? 

In this is all we need, can we add:
```suggestion
To get started, you need to install the `warp-ds/warp-ios` package

```